src/cli.ts

```diff
@@ -239,7 +239,7 @@ export function runWithOptions(options) {
         );
       })
       .then(function(lintWarnings){
-        lintWarnings.forEach(function(warning){
+        (lintWarnings as any[]).forEach(function(warning){
```

Comment:
This is the old, deprecated linter, it's using its own types. I wasn't confident what type was appropriate here.